hr_period/models/hr_fiscal_year.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+from openerp.tools.translate import _
+
+from datetime import datetime
+strftime = datetime.strptime
+
+from dateutil.relativedelta import relativedelta
+
+from itertools import chain
+
+INTERVALS = {
+    'annually': (relativedelta(months=12), 1),
+    'semi-annually': (relativedelta(months=6), 2),
+    'quaterly': (relativedelta(months=3), 4),
+    'bi-monthly': (relativedelta(months=2), 6),
+    'monthly': (relativedelta(months=1), 12),
+    'bi-weekly': (relativedelta(weeks=2), 26),
+    'weekly': (relativedelta(weeks=1), 52),
+}
```

Comment:
Is it ok to include 'daily' as well.
, there are countries that use daily workers, so at the end of the day to pay for it.